tmt/steps/provision/artemis.py

```diff
@@ -548,7 +548,22 @@ def _create(self) -> None:
         # TODO: spot instance
 
         if self.post_install_script:
-            data['post_install_script'] = self.post_install_script
+            if tmt.utils.is_url(self.post_install_script):
+                try:
+                    response = requests.get(self.post_install_script)
```

Comment:
Please use retry_session, for inspiration: https://github.com/teemtee/tmt/blob/1b6ac468c7ab26ca43b7c9ed006a48ce4e3dec01/tmt/steps/prepare/ansible.py#L127